index.html

```diff
@@ -0,0 +1,466 @@
+            <li class="mb-2">A Complete Protein Source Featuring 140mg of Tryptophan</li>
+            <li class="mb-2">Tasteless and Odorless or Available in Delicious Flavors</li>
+            <li class="mb-2">Mixes Instantly WITHOUT Clumps (in hot or cool liquids)</li>
+          </ul>
+        </ul>
+      </div>
+    </div>
+    <!--------------------------------------------------- Third Section------------------------------------------------------------------->
+    <div class="container-fluid">
+      <div class="row">
+        <div class="col-md-12">
+          <!--------------------------------------------------- Flex box------------------------------------------------------------------->
+          <div class="flex-container">
+            <!--------------------------------------------------- Logo Items------------------------------------------------------------------->
+            <div class="flex-item">
+              <div>
+                <div class="text-center">
+                  <img class="circle"
+                    src="https://cdn.shopify.com/s/files/1/0009/2900/5634/files/60-day-money-back-guarantee.png?33286"
+                    alt="60-Day Money Back Guarantee" width="50" height="49" />
+                </div>
+                <p class="text_review">60-Day Money Back Guarantee</p>
+              </div>
+            </div>
+            <div class="flex-item">
+              <div>
+                <div class="text-center"><img class="circle"
+                    src="https://cdn.shopify.com/s/files/1/0009/2900/5634/files/Complete_-5-Collagen-Protein.png?33316"
+                    alt="60-Day Money Back Guarantee" width="50" height="49" /></div>
+                <p class="text_review">COMPLETE, 5-COLLAGEN PROTEIN</p>
+              </div>
+            </div>
+            <div class="flex-item">
+              <div>
+                <div class="text-center"><img class="circle"
+                    src="https://cdn.shopify.com/s/files/1/0009/2900/5634/files/Hormone_-Glutena-and-Soy-Free_0cfd5c3d-c8a2-4659-ba48-bd631a05e7e5.png?347326"
+                    alt="60-Day Money Back Guarantee" width="50" height="49" /></div>
+                <p class="text_review">HORMONE & GLUTEN-FREE & NON-GMO</p>
+              </div>
+            </div>
+            <div class="flex-item">
+              <div>
+                <div class="text-center"><img class="circle mb-6"
+                    src="https://cdn.shopify.com/s/files/1/0009/2900/5634/files/never-clumps_609b1d6f-1c62-4cb9-8fe5-725c0dc114bb.png?34732"
+                    alt="60-Day Money Back Guarantee" width="50" height="49" /></div>
+                <p class="text_review">NEVER CLUMPS</p>
+              </div>
+            </div>
+          </div>
+
+          <!--------------------------------------------------- Customer Review Column------------------------------------------------------------------->
+          <div class="flexcontainer flexReviewSection">
```

Comment:
flexReviewSection is no being used anywhere?

Reply:
removed
`div now looks like
         <div class="customerReview">